shared/login/login/routes.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import * as I from 'immutable'
+import {RouteDefNode} from '../../route-tree'
+import Login from './'
+import UsernameOrEmail from '../register/username-or-email'
+import SelectOtherDevice from '../register/select-other-device'
+import Passphrase from '../register/passphrase'
+import PaperKey from '../register/paper-key'
+import CodePage from '../register/code-page'
+import SetPublicName from '../register/set-public-name'
+import Success from '../signup/success'
+import RegisterError from '../register/error'
+
+const recursiveLazyRoutes = I.Seq({
```

Comment:
This is a little different from the other route definitions. Because we don't know what order these screens will occur in, we need every child route to have the full set of children.